GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -59,6 +63,19 @@ public class Glia {
 
     var stringProvidingPhase: StringProvidingPhase = .notConfigured
 
+    var maximumUploads: Int {
+        var value = Self.maximumUploads
+        /// `MAXIMUM_UPLOADS_PER_MESSAGE` is used in acceptance tests to decrease the time
+        /// for uploading maximum number of files in a single message.
+        let env = ProcessInfo.processInfo.environment
```

Comment:
Is it possible to create managed dependency for ProcessInfo? This will allow you to test reading values from it via unit tests.

Reply:
Added 19d21ea308800f1cf9868494d88118003477e0ca